### ISSUE TITLE
[Idea] authlib-injector

### ISSUE THREAD

**Issue by RankFrank4010** [MCCTeam/Minecraft-Console-Client#2416]
### Prerequisites

- [X] I have read and understood the [user manual](https://mccteam.github.io/guide/)
- [X] I made sure I am running the latest [development build](https://github.com/MCCTeam/Minecraft-Console-Client/releases/latest)
- [X] I tried to [look for similar feature requests](https://github.com/MCCTeam/Minecraft-Console-Client/issues?q=is%3Aissue) before opening a new one

### Console Client Version

1.19.2

### Describe your problem

https://blessing.netlify.app/ I need to login a account with authlib-injector.

### Suggest a possible solution

[Chinese]在配置文件增加"Yggdrasil API::Yggdrasil API:"-""在AccountType中可以更改登录服务器为auth-injector。

### Device

Laptop

### Operating System

Windows

**Comment by milutinke** [comment]
Explain what is the idea in English.
If I understood this correctly, you want to use a custom authentication server to get some skins?

**Comment by RankFrank4010** [comment]
Yes

**Comment by RankFrank4010** [comment]
But the server must login with the account(Need authlib-injector)

**Comment by RankFrank4010** [comment]
But, i also want to use a custom authentication server to join server.(You must use the custom authentication server to join the server)

**Comment by RankFrank4010** [comment]
![authlib-injector server Yggdrasil API](https://user-images.githubusercontent.com/97322760/215247465-8b404a6f-707d-49b7-89d3-e36b9890a726.png)

**Comment by RankFrank4010** [comment]
![authlib-injector server Yggdrasil API](https://user-images.githubusercontent.com/97322760/215247849-25b4cb29-59d6-4295-86aa-f68df483026d.png)

**Comment by RankFrank4010** [comment]
But, i also want to use a custom authentication server to join server.(You must use the custom authentication server to join the minecraft server)

**Comment by RankFrank4010** [comment]
https://blessing.netlify.app/en 
Add Yggdrasil API:"-"(custom authentication server api), I can change AccountType to "authlib-injector"

**Comment by RankFrank4010** [comment]
Add Yggdrasil API:"-"(custom authentication server api), I can change AccountType to "authlib-injector"

**Comment by BruceChenQAQ** [comment]
Custom authentication server support may be added later (at least after #2385 is merged).

**Comment by RankFrank4010** [comment]
thank you very much

**Comment by RankFrank4010** [comment]
https://github.com/yushijinhun/authlib-injector/wiki/%E5%90%AF%E5%8A%A8%E5%99%A8%E6%8A%80%E6%9C%AF%E8%A7%84%E8%8C%83

**Comment by milutinke** [comment]
@RankFrank4010 

Join our discord https://discord.com/invite/sfBv4TtpC9
We can work it out there.
I would need access to the API and the panel for testing

**Comment by GLDYM** [comment]
Any future works?

**Comment by milutinke** [comment]
> Any future works?
> 

We need someone to setup a test instance of the auth server so we can add and test this.
If you're willing to help with that, hit me up on the discord server in like a week.

**Comment by RankFrank4010** [comment]
I will

**Comment by RankFrank4010** [comment]
I send a minecraft server (use a public auth server to auth)test instance in the discord server and I hope the client will be great

**Comment by GLDYM** [comment]
Already done.
